docs/INSTALLATION.md

```diff
@@ -0,0 +1,265 @@
+# Installing OVE
+
+OVE needs to be installed before it can be used to control a display. OVE can be installed either by downloading and compiling the source code of the corresponding components or by running a specific installer available on the [**OVE Install**](https://github.com/ove/ove-install/) repository.
+
+All contributors to OVE are encouraged to download and compile the source code. All users of OVE are encouraged to use the OVE installers.
+
+## Installation by running OVE installers
+
+[**OVE Install**](https://github.com/ove/ove-install/) scripts are designed to install OVE into a [Docker](https://www.docker.com/get-started) environment.
+
+### Prerequisites
+
+* [git](https://git-scm.com/downloads)
+* [Docker](https://www.docker.com/get-started)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Configuring the environment
+
+The deployment environment needs to be pre-configured before running the OVE installers. This involves setting up ports and environment variables.
+
+The ports are pre-configured to a list of common defaults, but can be changed based on end-user requirements. Each port or port-range is defined as a mapping `HOST_PORT:CONTAINER_PORT`. Only the host ports can be changed, and it is important to note that **container ports must not be changed**.
+
+#### Environment variables
```

Comment:
I find it odd that we tell people how to run `docker ps` however do not tell them how to set an Environmental variable... especially as setting the host OS environmental variables will **not** work

If using Docker-compose, these are edited in the .yml files